File: ecommerc_dotnet/dto/Response/SubCategoryResponseDto.cs
using ecommerc_dotnet.module;

namespace ecommerc_dotnet.dto.Response;

public class SubCategoryResponseDto
{
    public Guid id { get; set; }
    public string name { get; set; }
    public Guid categoryId { get; set; } 
    public Guid storeId { get; set; } 
    
}

File: eccorce_app/app/src/main/java/com/example/eccomerce_app/dto/response/SubCategoryResponseDto.kt
package com.example.e_commercompose.dto.response

import com.example.hotel_mobile.services.kSerializeChanger.UUIDKserialize
import kotlinx.serialization.Serializable
import java.util.UUID

@Serializable
data class SubCategoryResponseDto(
    val name: String,
    @Serializable(with = UUIDKserialize::class)
    val id: UUID,
    @Serializable(with = UUIDKserialize::class)
    val categoryId: UUID,
    @Serializable(with = UUIDKserialize::class)
    val storeId: UUID
)
